docs/release-notes/deprecate-k8s-preemption-scheduler.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Deprecations**
+
+-  Kubernetes Scheduling: Support for the priority with preemption scheduler for Kubernetes Resource Managers
```

Comment:
Please add a note telling the user what to use instead or what to do about this. Remove the config line? Update to something else?

Reply:
done!